### PR TITLE
feat(AnimePahe): Poster as largeImageKey

### PR DESCRIPTION
**Pull request by MulverineX**
image
image

### CODE REVIEW

**Comment by MulverineX** [comment]
shit I screwed it up. greeaaaat

File: websites/A/animepahe/presence.ts [hunk 1]
```diff
@@ -274,6 +274,12 @@ presence.on("UpdateData", async () => {
 
                 presenceData.state = title;
 
+                presenceData.largeImageKey = (
+                  document.getElementsByClassName(
+                    "youtube-preview"
+                  )[0] as HTMLAnchorElement
```
Comment (Bas950) [inline websites/A/animepahe/presence.ts:280]:
```suggestion
                  document.getElementsByClassName<HTMLAnchorElement>(
                    "youtube-preview"
                  )[0]
```

File: websites/A/animepahe/presence.ts [hunk 2]
```diff
@@ -319,6 +325,10 @@ presence.on("UpdateData", async () => {
 
         presenceData.state = title;
 
+        presenceData.largeImageKey = (
+          document.getElementsByClassName("anime-poster")[0] as HTMLImageElement
+        ).src.replace(".th", "");
```
Comment (Bas950) [inline websites/A/animepahe/presence.ts:330]:
```suggestion
        presenceData.largeImageKey = (
          document.getElementsByClassName<HTMLImageElement>("anime-poster")[0]
        ).src.replace(".th", "");
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — DISMISSED** [review]

**Comment by MulverineX** [comment]
@Bas950 the typescript compiler does not like that

**Comment by MulverineX** [comment]
image

**Comment by Bas950** [comment]
> image image

document.querySelectorAll<HTMLAnchorElement>(
                    ".youtube-preview"
                  )[0].href;
                  Would that work?

**Comment by MulverineX** [comment]
`querySelector` is awful but yes this works. 
image

**Comment by Bas950** [comment]
> `querySelector` is awful but yes this works. image
> 
> Do I have to do this?

I like it better than the getElementByClassname + as something

**Comment by MulverineX** [comment]
I don't like it but I did it, ready to merge. @Bas950

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**